servicecontrol/license.md

```diff
@@ -30,6 +34,12 @@ To import a license using PowerShell:
 Import-ServiceControlLicense <LicenseFile>
 ```
 
+#### Imported locations
+ 
+The license is then imported at *the machine level* in the registry (`HKLM\SOFTWARE\ParticularSoftware\@License`) and on the file system (`%ProgramData%\ParticularSoftware\license.xml`).
```

Comment:
Did you verify that the powershell script writes to the file system also? That wasn't a change that I was aware of.

Reply:
Also, same question about "the machine level" formatting.

Reply:
@bording I looked at the SC repo, reviewed the cmdlet and it is using the `LicenseManager` from `ServiceControlInstaller.Engine` which does both a registry and filesystem update.

Reply:
@ramonsmits Cool, I wasn't aware that it reused the class. 

In that case, I'm wondering if it makes sense to restructure things so that there only needs to be one "Imported locations" section vs. having the same content duplicated.